src/main/java/heronarts/lx/audio/LXAudioComponent.java

```diff
@@ -52,6 +52,8 @@
   public final LXAudioBuffer right = new LXAudioBuffer(SAMPLE_BUFFER_SIZE, SAMPLE_RATE);
   public final LXAudioBuffer mix = new LXAudioBuffer(SAMPLE_BUFFER_SIZE, SAMPLE_RATE);
 
+  protected boolean stopped = false;
```

Comment:
I don't think this helps. The variable is never referenced in LXAudioComponent, but it has now been downgraded from `private` to `protected` - which opens it up to being touched in ways that could introduce bugs or side-effects in the subclasses.

Each subclass used its `stopped` variable totally internally, which is why they were both `private`